app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def update(
+            self,
+            id_to_update: int,
+            first_name_upd: str,
+            last_name_upd: str
+    ) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            f"SET (first_name, last_name) = (?, ?) "
```

Comment:
The SQL syntax in the `update` method is incorrect. The `SET` clause should specify each column separately, like `SET first_name = ?, last_name = ?`.